esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -134,11 +134,14 @@ struct WsWUExceptions
 #define WUINFO_IncludeHelpers           0x8000
 #define WUINFO_IncludeAllowedClusters   0x10000
 #define WUINFO_IncludeTotalClusterTime  0x20000
-#define WUINFO_IncludeSOAPDatasetNode   0x40000
+#define WUINFO_IncludeServiceActivities 0x40000
 #define WUINFO_All                      0xFFFFFFFF
 
 class WsWuInfo
 {
+    std::set<unsigned> serviceActivities;
```

Comment:
Very picky, but the name "serviceActivities" could be kinds or nodes.. maybe call this "serviceActivityKinds"?